src/NHSD.BuyingCatalogue.Identity.Api/Controllers/ErrorController.cs

```diff
@@ -21,7 +21,7 @@ public class ErrorController : Controller
         public async Task<IActionResult> Index(string errorId)
         {
             var message = await _interactionService.GetErrorContextAsync(errorId);
-            _logger.Log(LogLevel.Error, "Request for client {0} failed with error code {1}: {2}",
+            _logger.Log(LogLevel.Error, "Request for client {ClientId} failed with error code {Error}: {ErrorDescription}",
```

Comment:
Did we agree here to use the _logger.LogError extension method rather than the concrete implementation?